src/util/system.cpp

```diff
@@ -660,13 +660,11 @@ std::string PrivacyInfo()
 
 std::string LicenseInfo()
 {
-    return "\n" +
-           FormatParagraph(strprintf(_("Copyright (C) 2009-%i The Bitcoin Core Developers"), COPYRIGHT_YEAR)) + "\n" +
-           FormatParagraph(strprintf(_("Copyright (C) 2015-%i The Zcash Developers"), COPYRIGHT_YEAR)) + "\n" +
+    return CopyrightHolders(strprintf(_("Copyright (C) %i-%i"), 2009, COPYRIGHT_YEAR) + " ") + "\n" +
```

Comment:
This changes the copyright year range to start at 2009 for both The Zcash Developers and The Bitcoin Core Developers. But the Zcash developers didn't exist as such in 2009; 2015 is more correct for that line.